Gemfile.lock

```diff
@@ -835,4 +829,4 @@ RUBY VERSION
    ruby 3.0.3p157
 
 BUNDLED WITH
-   2.1.4
+   2.2.32
```

Comment:
You are using a different bundler version here. Can you keep it the same for now? Upgrading bundler would be a different pull request.